Trivago/Jade/Application/Framework/JadeBundle/Controller/JsonApiController.php

```diff
@@ -228,18 +228,25 @@ protected function getCollection($resourceName, Request $httpRequest)
 
         if ($this->getParameter('json_api_fetch_total_count')) {
             if (!$repository instanceof ResourceCounter) {
-                throw new \LogicException(sprintf(
-                    'You have enabled fetch_total_count option. The repository for %s should implement ResourceCounter',
-                    get_class($repository)
-                ));
+                throw new \LogicException(
+                    sprintf(
+                        'You have enabled fetch_total_count option. The repository for %s should implement ResourceCounter',
+                        get_class($repository)
+                    )
+                );
             }
             $data['meta']['totalItems'] = $repository->countResourcesWithFilters(
                 $request->getRelationships(),
                 $request->getConstraint()->getFilterCollection()
             );
 
-            $data['meta']['totalPages'] = (int) (abs($data['meta']['totalItems']-1)/$request->getConstraint()->getPerPage() + 1);
-            $data['links'] = $this->buildLinks($httpRequest, $request->getConstraint()->getPageNumber(), $data['meta']['totalPages']);
+            $data['meta']['totalPages'] = (int) (abs($data['meta']['totalItems'] - 1) / $request->getConstraint(
```

Comment:
This formatting looks way too forced
Maybe you can move the calculation out to a function and that should fix the formatting also makes it more readable